src/main/resources/stairway/db/changesets/20210416_flightworking.yaml

```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: flightworking
+      author: JCz
+      changes:
+        - dropNotNullConstraint:
+            tableName: flightlog
+            columnName: working_parameters
+            remarks: deprecating for flightworking table
+        - addColumn:
+            tableName: flightlog
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
+                    unique: true
+        - createTable:
+            tableName: flightworking
```

Comment:
optional: maybe `flightworkingparameters` or `flightworkingmap`? `flightworking` feels like it could be almost anything.

Reply:
Unless @ddietterich has a preference for something else, I'll stick with his original naming.

Reply:
I think it is fine the way it is, FWIW